test/repository_test.rb

```diff
@@ -117,6 +118,34 @@
         end
       end
 
+      describe '.created_at' do
+
+        before do
+
+          @user = UserRepository.create(user1)
+          @timeago = DateTime.new(2015,4,1)
+        
+        end
+
+        let(:article4) { Article.new(user_id: @user.id, title: 'Love Relationships', comments_count: '4') }
```

Comment:
Firstly, what do you think about using a simpler fixture entity? Something that has no association, for example User

Secondly, naming with number suffix does not give reader the context of fixtures. What do you think about naming it like  `article_without_created_at` and `article_with_created_at`?